src/processes.md

```diff
@@ -0,0 +1,14 @@
+# Questions about your development processes
+
+## How should I use tools differently from C++?
+
+* *Use `rustfmt` automatically everywhere.* While in C++ there are many
+  different coding styles, the Rust community is in agreement (at least,
+  they're in agreement that it's a good idea to be in agreement). That
+  is codified in `rustfmt`. Use it, automatically, on every submission.
+* *Use `clippy` somewhere*. Its lints are useful.
+* *Use IDEs more liberally*. Even staunch vim-adherents (your author!)
```

Comment:
I haven't tried it yet, but there is https://github.com/rust-lang/rust.vim and similar things.

Reply:
Yeah. I think it's too much detail for the FAQ, and I also have not tried it, but I'm inclined to suspect it's not actually as good as a full-featured graphical IDE. I think the tiny fonts of a GUI are useful for type annotations. I might be wrong though, I'll give it a try sometime!